stable/UpgradeManifestV1.json

```
@@ -1 +1 @@
-{"py/object": "pros.upgrade.manifests.upgrade_manifest_v1.UpgradeManifestV1", "info_url": "https://pros.cs.purdue.edu/v5/releases/cli3.2.3.html", "version": {"py/object": "semantic_version.base.Version", "build": {"py/tuple": []}, "major": 3, "minor": 2, "partial": false, "patch": 3, "prerelease": {"py/tuple": []}}}
+{"py/object": "pros.upgrade.manifests.upgrade_manifest_v1.UpgradeManifestV1", "info_url": "https://pros.cs.purdue.edu/v5/releases/cli3.3.3.html", "version": {"py/object": "semantic_version.base.Version", "build": {"py/tuple": []}, "major": 3, "minor": 3, "partial": false, "patch": 3, "prerelease": {"py/tuple": []}}}
```

```suggestion
{"py/object": "pros.upgrade.manifests.upgrade_manifest_v1.UpgradeManifestV1", "info_url": "https://pros.cs.purdue.edu/v5/releases/cli3.3.3.html", "version": {"py/object": "semantic_version.base.Version", "build": {"py/tuple": []}, "major": 3, "minor": 3, "partial": false, "patch": 3, "prerelease": {"py/tuple": []}}}
```
Cli 3.3.3 blog release post (the url included) does not exist, need to make that first.